wallet/src/bin/faucet_server.rs

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2022 Espresso Systems (espressosys.com)
```

Comment:
Should we add a `faucet` crate in the workspace or is that somehow inconvenient?